jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -967,6 +970,8 @@ def _load_regions(self, regions, combination_mode=None, max_num_regions=None,
             n_loaded += 1
             if max_num_regions is not None and n_loaded >= max_num_regions:
                 break
+        # Revert edit mode to before the import_region call
+        self.app.session.edit_subset_mode.mode = previous_mode
```

Comment:
is there anything above that could actually raise an error (not caught by `bad_regions`) where the revert should still happen?

Reply:
Not that I could see, does anything stick out to you?

Reply:
I guess we don't need to catch bugs in the code itself, so `bad_regions` should be sufficient to make sure the revert happens here.  Thanks!